kill-stuck-pr-test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+source ${CMS_BOT_DIR}/jenkins-artifacts
+rm -f *.prop
+
+if [ "X${UPLOAD_UNIQUE_ID}" = "X" ] ; then exit 0 ; fi
+if [ "X${PULL_REQUEST}" = "X" ] ; then exit 0 ; fi
+
+REPOSITORY=$(echo ${PULL_REQUEST} | cut -d '#' -f 1)
+PR_ID=$(echo ${PULL_REQUEST} | cut -d '#' -f 2)
+
+COMMIT_ID=$(grep ${ARTIFACT_BASE_DIR_MAIN}/pull-request-integration/${UPLOAD_UNIQUE_ID}/prs_commits.txt "^${PULL_REQUEST}=")
```

Comment:
@iarspider , `grep` command does not look correct, I guess the file name should be at the end